docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/heroku.md

```diff
@@ -287,116 +110,187 @@ module.exports = ({ env }) => ({
 <code-block title="TYPESCRIPT">
 
 
-```js
+```ts
+// Path: ./config/env/production/server.js`
 export default ({ env }) => ({
-  url: env('MY_HEROKU_URL'),
-});
+        proxy: true,
+        url: env('APP_URL'), // replaces `host` and `port` properties in the development environment
+        app: { 
+          keys: env.array('APP_KEYS')
+        },
+    });
 ```
 
 </code-block>
 </code-group>
 
+6. Add PostgreSQL dependencies by installing [`pg` package](https://www.npmjs.com/package/pg) and [`pg-connection-string` package](https://www.npmjs.com/package/pg-connection-string):
 
+    <code-group>
 
-You will also need to set the environment variable in Heroku for the `MY_HEROKU_URL`. This will populate the variable with something like `https://your-app.herokuapp.com`.
+    <code-block title="NPM">
+    ```sh
+    npm install pg && npm install pg-connection-string
+    ```
+    </code-block>
 
-```bash
-heroku config:set MY_HEROKU_URL=$(heroku info -s | grep web_url | cut -d= -f2)
-heroku config:set APP_KEYS=$(cat .env | grep APP_KEYS | cut -d= -f2-)
-heroku config:set API_TOKEN_SALT=$(cat .env | grep API_TOKEN_SALT | cut -d= -f2)
-heroku config:set ADMIN_JWT_SECRET=$(cat .env | grep ADMIN_JWT_SECRET | cut -d= -f2)
-heroku config:set JWT_SECRET=$(cat .env | grep -w JWT_SECRET | cut -d= -f2)
-```
+    <code-block title="YARN">
+    ```sh
+    yarn add pg && yarn add pg-connection-string
+    ```
+    </code-block>
 
-:::tip
-On Windows, variables can be set manually by running the `heroku config:set VARIABLE=your-key-here` for each variable.
-:::
+    </code-group>
 
-The following `openssl` commands will generate random new secrets (Mac and Linux only):
+7. Add `package.json` to the end of the `.gitignore` file at the root of your Strapi project:
```

Comment:
package-lock.json not package.json. If you ignore the package.json then heroku won't know how to install packages.

Same for the yarn.lock file

Reply:
ugh, that was a bad typo. Thanks for catching it.